tests/installation/select_packages.pm

```diff
@@ -66,7 +66,7 @@ sub run {
     }
 
     # Check that installation is blocked (expected behavior)
-    assert_screen 'inst-overview-blocked';
+    return unless check_screen 'inst-overview-blocked';
```

Comment:
The **assert_screen** is important. The test is expecting to test a behavior from yast overview to block the installation if a required package i.e. & e.g. grub2 is not selected. See https://openqa.suse.de/tests/5443793#step/select_packages/44 (please, add verification run for that scenario too)

Removing the assert screen and returning will cause false negatives when a regression happens.

If the intention is to misuse this module for different purposes, the last part should be indented inside an **if** which allows that part to be **explicitly skipped** with a job setting.

For example,

```
if ($blocker_packages) {
 ...
}
```

Which comes from `my $blocker_packages = get_var('INSTALLATION_BLOCKED');`
Which is set as `INSTALLATION_BLOCKED=grub2`